examples/gallery/line/README.txt

```diff
@@ -1,2 +1,2 @@
-Lines
------
+Lines and vectors
```

Comment:
@willschlitzer Need to rename the directory "line" to "lines"

Reply:
Done in 3639b50. Looks like PyCharm had some questions about what to change when I hit "refactor" and I never clicked through.